### ISSUE TITLE
Could rwgps be broken

### ISSUE THREAD

**Issue by lewisl** [cpfair/tapiriik#392]
I receive this in the log:
Ride With GPS: There was a problem indexing your activities on Ride With GPS, so no activities will be uploaded to Ride With GPS.   By the way I have restricted the date range to 1 jan 2017 to minimize activities.

I can't sync to rwgps from either garmin connect or dropbox.  Nothing special in either case--I did have to "unprivate" everything in garmin connect, but after I did so everything synced nicely to dropbox.  Can't get anything into rwgps.

Sent a message to them but they (naturally) say talk to Collin, he is really great....  (And so he is and this is a lot of work.)   I re-pinged rwgps suggesting they could look a bit harder.

**Comment by colecallahan** [comment]
@lewisl i'm having the same issue re:RWGPS.  i just finished chatting with them and they said they don't have any bearing on how tapiriik works, it just pulls data from them.  they have reached out to collin, but haven't heard yet.  hopefully it gets fixed soon, i'm not sure i can survive without kudos.

**Comment by rnc** [comment]
I have seen this over, possibly the past week (?). I've been uploading my rides manually but its a right pain ;-)

**Comment by colecallahan** [comment]
I don’t know what happened but it all synced last night while I was asleep. Did you have any luck?

**Comment by rnc** [comment]
@colecallahan  Yeah agreed. Its worked this morning and it now knows about all activites again. 
@lewisl  Is it now working for you?

**Comment by lewisl** [comment]
Yeah.  This morning dozens of old rides had synced.  

Yesterday, I did change all my settings in rwgps to public (rather than friends--hate doing that).  I thought maybe that change fixed things but looks like coincidence as something else happened.

I am going to go back to "friends" and see what happens as I am only syncing inbound to rwgps.

**Comment by campbellr** [comment]
Looks like it must have been fixed by a52cda5, which Collin just pushed in yesterday.

**Comment by rnc** [comment]
Nice! Thanks @cpfair

**Comment by monaxle** [comment]
Ride With GPS: There was a problem indexing your activities on Ride With GPS, so no activities will be uploaded to Ride With GPS. 

@cpfair This problem has come up for me on each ride completed over the last 6 days and on sporadic rides over the last month or so. Few other rides over the last 2 -3 months not synced to other services connected for reasons that are not clear to me. I never change the settings on my accounts. Is this something fixable? Be a shame to lose the service. I wondered whether my subscription was over due but can;t see anything to suggest it is.
